src/unity/Assets/Crest/Scripts/OceanDepthCache.cs

```diff
@@ -40,6 +49,9 @@ public void PopulateCache()
                 _drawCacheQuad = GameObject.CreatePrimitive(PrimitiveType.Quad);
                 Destroy(_drawCacheQuad.GetComponent<Collider>());
                 _drawCacheQuad.name = "Draw_" + _cache.name;
+                Vector3 pos = _drawCacheQuad.transform.position;
+                pos.y = 0; //force it to be at 0 height so the quad can be seen by the water cameras
+                _drawCacheQuad.transform.position = pos;
```

Comment:
I'm a bit unsure about this one - I give false as a second argument to set transform on the line below which i thought was actually not taking the world position. can you confirm that this code is merged up correctly and fixes an issue?